src/Storage/FilenameStrategyInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Storage;
+
+interface FilenameStrategyInterface
+{
+    /**
+     * Generate dump file name.
+     */
+    public function generateFilename(
+        string $storageRootDir,
```

Comment:
Is the storage root directory really necessary? Could we not just expect a filename relative to this root from implementations?